networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/GossipFailureLoggerTest.java

```diff
@@ -43,7 +43,7 @@ void shouldLogAlreadySeenErrorsAtDebugLevel() {
   void shouldLogFirstNoPeersErrorsAtWarningLevel() {
     try (final LogCaptor logCaptor = LogCaptor.forClass(GossipFailureLogger.class)) {
       logger.logWithSuppression(
-          new RuntimeException("Foo", new NoPeersForOutboundMessageException("So Lonely")), SLOT);
+          new RuntimeException("Foo", new SemiDuplexNoOutboundStreamException("So Lonely")), SLOT);
```

Comment:
this SemiDuplex exception used to share implementation with NoPeers error handling so this still tests the same thing and all the logic continues to work.